assemblies/pentaho-war/pom.xml

```diff
@@ -29,6 +29,7 @@
     <replacer.version>1.5.2</replacer.version>
     <rxjava.version>2.2.3</rxjava.version>
     <snowflake-jdbc.version>3.6.28</snowflake-jdbc.version>
+    <big-data-plugin.version>9.1.0.0-SNAPSHOT</big-data-plugin.version>
```

Comment:
If this is always the same as the project version, better not use a separate property or it will not get updated by the build system.